tests/test_job.py

```diff
@@ -1064,8 +1064,7 @@ def test_reset_statepoint_project(self):
         src_job.data[key] = d
         assert key in src_job.data
         assert len(src_job.data) == 1
-        with pytest.deprecated_call():
```

Comment:
This test is now an exact duplicate of `test_reset_statepoint_job`. I removed it.